docs/docs/compatibility-matrix.mdx

```diff
@@ -0,0 +1,19 @@
+---
+id: compatibility-matrix
+sidebar_label: Compatibility Matrix
+title: Compatibility Matrix
+description: |
+  Information about compatibility between Rasa Pro Services and Rasa Plus.
+---
+
+
+When choosing a version of Rasa Pro Services to [deploy](./deploy/deploy-rasa-pro-services.mdx), you can refer to the
+table below to see which one is compatible with your installation of Rasa Plus.
+
+Rasa Pro Services version is independent of Rasa Plus version, except that they share the same major version number.
+
+
+| Rasa Pro Services | Rasa Plus |
```

Comment:
This doesn't render correctly in the preview for some reason 🤔

Reply:
yeah I think the theme tables are broken everywhere... will see if we can address this before the release (I am not sure we can)